abci/types/types.proto

```diff
@@ -271,6 +272,10 @@ message Evidence {
   int64 total_voting_power = 5;
 }
 
+message Event {
```

Comment:
"type" or "name" that is a string first, before tags.  Maybe rename tags to attributes or attrs.